### PR TITLE
Adding instructions for a shallow clone in the CONTRIBUTING page

### PR DESCRIPTION
**Pull request by joshmgrant**
### Description
For faster development, it's recommended to use a shallow clone (`--depth 1`) instead of a full clone. I have updated the CONTRIBUTING doc to mention this.

### Motivation and Context
Every little bit helps. If a new or existing contributor can get the Selenium code base in a clone operation more quickly, they may see a better chance to make a change and open a PR. This change helps these new contributors get up and started quickly.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by AutomatedTester** [comment]
I’ve not gone through the readme but there are a lot of changes in this PR for a readme change. Can you check and rebase accordingly

David

**Comment by joshmgrant** [comment]
@AutomatedTester sorry about that, PR is now in better shape

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,9 +68,10 @@ Fork the project [on Github](https://github.com/seleniumhq/selenium)
 and check out your copy locally.
 
 ```shell
-% git clone git@github.com:username/selenium.git
+% git clone --depth 1 git@github.com:username/selenium.git # shallow clone recommended
```
Comment (titusfortner) [inline CONTRIBUTING.md:71]: I think the default should be to clone the whole thing, but it does make sense to add a note after this code snippet that if you do not need commit history, using `--depth 1` will be much faster.

Comment (diemol) [reply CONTRIBUTING.md:71]: @joshmgrant I think what Titus says makes sense, feel free to make the changes and then we can merge.

**Comment by joshmgrant** [comment]
After some consideration, I don't think this PR is meaningful. Thanks!

**Pull request closed without merging**